docs/production-deployment/cloud/aws-export-s3.mdx

```diff
@@ -0,0 +1,134 @@
+---
+id: aws-export-s3
+title: Exporting Workflow Event History to AWS S3
+sidebar_label: AWS Export to S3
+description: Export Workflow History to AWS S3
+slug: /cloud/export/aws-export-s3
+toc_max_heading_level: 4
+keywords:
+  - explanation
+  - how-to
+  - operations
+  - temporal cloud
+  - term
+  - verify
+  - workflow history export
+  - aws
+tags:
+  - explanation
+  - how-to
+  - operations
+  - temporal-cloud
+  - term
+  - verify
+  - workflow-history-export
+  - aws
+---
+
+### Prerequisites {#prerequisites}
+
+Before configuring the Export Sink, ensure you have the following:
+
+1. An AWS Account with write permission to an S3 bucket.
+2. An AWS S3 bucket.
+   1. The S3 bucket must reside in the same region as your Namespace.
+3. (optional) A KMS ARN associated with the S3 bucket.
+
+## Configure Workflow History Export 
+
+**How to configure Workflow History Export**
+
+You can use either the [Temporal Cloud UI](#using-temporal-cloud-ui) or [tcld](#using-tcld) to configure the Workflow History Export.
+
+The Temporal Cloud UI provides two ways for configuring Workflow History Export:
+
+- [Automated setup](#automated-setup) (recommended): The Cloud UI launches the AWS CloudFormation Console to create a stack, with write permission to the S3 bucket.
+- [Manual setup](#manual-setup): The Cloud UI provides a CloudFormation template for users to manually configure a CloudFormation stack.
+
+### Using Temporal Cloud UI
+
+The following steps guide you through setting up Workflow History Export using the Temporal Cloud UI.
+
+![](/img/export-sink-ui.png)
+
+:::tip
+
+Don't forget to click Create at the end of your setup to confirm your export.
+
+:::
+
+#### Automated setup
+
+The automated setup creates a CloudFormation stack with write permission to the S3 bucket.
+[Verify the export setup](#verify) before saving the configuration.
+
+1. Open the Temporal Cloud UI and navigate to the Namespace you want to configure.
+2. Select **Configure** from the **Export** card.
+3. Provide the following information to configure the export sink and then select **Create and launch stack**:
+   1. Name: A name for the export sink.
```

Comment:
These can be dashes. I don't believe we usually do numericals after a numerical.

Reply:
Fixed!